bin/qtui-rebind

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+"""
+    Simple script to rewrite Qt ui files to change shortcut bindings.
```

Comment:
When we're working on projects, our personal stance isn't really relevant. Except in extreme circumstances we must fit in with what's already there, even if we disagree with it.